### ISSUE TITLE
PROPOSAL - Pre-fill previous answers

### ISSUE THREAD

**Issue by ptr0x1** [standup-raven/standup-raven#178]
**Is your feature request related to a problem? Please describe.**
Our day-to-day work often involves week-long projects where much of the same information is added day-to-day (with some changes).

**Describe the solution you'd like**
It would be great to have a 'fill yesterday's answer' next to the Save button, which would allow people to more quickly edit only what has changed and keep the rest.

**Describe alternatives you've considered**
We've thought about switching to a weekly standup but it is nice to have a daily routine of starting the day with the standup.

**Additional context**
I think this could be doable by loading in the last answers for each user and pre-filling the fields I would be happy to help with it if you think this is worthwhile addition (although I'm not that familiar with Go+React yet).

**Comment by harshilsharma63** [comment]
@ptr0x1 thanks a lot for the feature request!

We actually have this on the mind and other users have requested this as well.

The complexity of this feature is that those sections are custom. For example, I could have a standup config where I have no section for yesterday's task. Similarly, the name of the section can be anything, for example - "yesterday", "previous day" etc.  Also, there can be multiple sections that logically denote yesterday's tasks. So I'm wondering how would that work.

One solution is that while defining the sections, we can have a checkbox next to them indicating that this section data should be pre-loaded in the next day's standup.

What do you think of this approach?

**Comment by ptr0x1** [comment]
Ah that's a very good point that I did not think about - that custom questions complicates it quite a bit. I think that pre-load check is a great idea. Maybe it could also have an extra option to 'move answers to the next/previous question'? 

Our questions relate to things you 1. did yesterday, 2. plan on doing today, and 3. blockers. So most of the time on day1 you set what you do today, then on day2 those answers will be what you did yesterday (provided you finished them, but it's easier to erase than putting in the same thing). I don't know how custom this use-case is though.

**Comment by harshilsharma63** [comment]
@ptr0x1 yes I agree this will make it easier to fill standups. I like `move answers to the next/previous question` idea. Or maybe for simplicity, we always move it to the next section. I think it would work. Just need a simple enough UI for it when configuring.
